### PR TITLE
Change idle handling (add 6 min shut off & increase normal idle wait)

### PR DESCRIPTION
**Pull request by helloworld111gh**
This PR will turn off the extension if the user is idle for the period of ~~800 seconds~~ 6 minutes (45x the non-idle polling rate)

This will give more work to active users and reduce work given to inactive users by ensuring idle testers do not poll for new jobs.

The PR also increases the time needed till the extension goes into idle mode at 24 seconds (3 requests).

### CODE REVIEW

**Comment by ukd1** [comment]
@helloworld111gh this actually looks really good insight and made me think. Your code is fine and I would merge it...but...I think we should do this differently: if someone is idle, we should set ``_checking_active = false``, which would mean the new tab would never open. Is there a legit reason to be idle AND expect work?

Thoughts?

**Comment by helloworld111gh** [comment]
because someone can not use the keyboard/mouse for 15/24 second but still be ready (watching video, reading news article), but it can be more certain if they done nothing for a few minutes.
I've reduced the turn-off wait to 6 minutes, but if you want you can adjust it

File: extension/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Rainforest Tester Notifier [DEV]",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (ukd1) [inline extension/manifest.json]: @helloworld111gh can you revert this change please?

File: production_manifest.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Rainforest Tester Notifier",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (ukd1) [inline production_manifest.json]: @helloworld111gh can you revert this too please?

File: staging_manifest.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Rainforest Tester Notifier [STG]",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (ukd1) [inline staging_manifest.json]: @helloworld111gh and this :)

**Comment by juliankrispel** [comment]
awesome thanks @helloworld111gh
